Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -454,6 +454,7 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             // Jobs
             routes.MapHttpWebJobsRoute("list-all-jobs", "", "", new { controller = "Jobs", action = "ListAllJobs" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpWebJobsRoute("list-triggered-jobs", "triggered", "", new { controller = "Jobs", action = "ListTriggeredJobs" }, new { verb = new HttpMethodConstraint("GET") });
+            routes.MapHttpWebJobsRoute("list-triggered-jobs-swagger", "triggered", "/swagger", new { controller = "Jobs", action = "ListTriggeredJobsInSwaggerFormat" }, new { verb = new HttpMethodConstraint("GET") });
```

Comment:
Are you sure it can't have a query string? That would indeed be cleaner. Need to check with @mikestall